src/databases/index.ts

```diff
@@ -96,3 +96,28 @@ export async function getQueryCachePath(
   const cacheName = `${await nameHash(databaseName)}-${await hash(strings, ...params)}.json`;
   return join(sourceDir, ".observable", "cache", cacheName);
 }
+
+/**
+ * Reads partial database configurations from the .observable/databases.json file.
+ * For security reasons, no identifiers or passwords are returned.
```

Comment:
Not sure this redaction accomplishes anything, since any process can read the .observable/databases.json file directly. I guess if we want to call this `getDatabaseTypes` then we could have a `Map<string, string>` that tells you the datbase types… but it’s probably just fine to load the unredacted database configs, too.